index.html

```diff
@@ -22,16 +22,18 @@
 		<header id="header"></header>
 
 		<div class="homeContent">
-		<br>
+			<br>
+			<div class="homeAlert" id="homeAlert">
+				<p>Interested in applying to The Forum Helpers? Check out our <a class="FHULink" href="apply">how to apply page</a>.</p>
```

Comment:
not sure about grammer here. thoughts?

Reply:
seems alright to me.

Reply:
okay. wanna resolve this someone? i can't